Simplified/LibraryFinder/UI/NYPLLibraryFinderViewController.swift

```diff
@@ -0,0 +1,257 @@
+      let account = dataProvider.newLibraryAccounts[indexPath.item]
+      cell.configureCell(type: .newLibrary, account: account)
+    }
+    return cell
+  }
+  
+  // MARK: - FlowLayout
+  
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+    return section == NYPLLibraryFinderSection.myLibrary.rawValue ? 0 : 8
+  }
+  
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+    if section == NYPLLibraryFinderSection.newLibrary.rawValue && dataProvider.newLibraryAccounts.count == 0 {
+      return CGSize.zero
+    }
+    return CGSize(width: collectionView.bounds.width - NYPLLibraryFinderConfiguration.collectionViewContentInset * 2, height: 50)
+  }
+  
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForFooterInSection section: Int) -> CGSize {
+    var height: CGFloat = 0.0
+    if section == NYPLLibraryFinderSection.myLibrary.rawValue {
+      height = 10.0
+    }
+    return CGSize(width: collectionView.bounds.width - NYPLLibraryFinderConfiguration.collectionViewContentInset * 2, height: height)
+  }
+  
+  override func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+    if indexPath.section == NYPLLibraryFinderSection.searchBar.rawValue {
+      let view = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: searchBarReuseIdentifier, for: indexPath)
+      view.addSubview(searchBar)
+      searchBar.autoPinEdgesToSuperviewEdges(with: UIEdgeInsets(top: 0, left: 0, bottom: 10, right: 0))
+      return view
+    }
+    
+    if kind == UICollectionView.elementKindSectionHeader,
+      let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind,
+                                                                   withReuseIdentifier: NYPLLibraryFinderSectionHeaderView.reuseIdentifier,
+                                                                   for: indexPath) as? NYPLLibraryFinderSectionHeaderView {
+      let type: NYPLLibraryFinderLibraryCellType = indexPath.section == NYPLLibraryFinderSection.myLibrary.rawValue ? .myLibrary : .newLibrary
+      header.configureView(type: type, displayer: self)
+      return header
+    }
+    
+    if kind == UICollectionView.elementKindSectionFooter,
+      let footer = collectionView.dequeueReusableSupplementaryView(ofKind: kind,
+                                                                   withReuseIdentifier: NYPLLibraryFinderSectionFooterView.reuseIdentifier,
+                                                                   for: indexPath) as? NYPLLibraryFinderSectionFooterView {
+      return footer
```

Comment:
I think we can simply ignore the assignment statement here unless you intend to keep it here for debugging purposes